aodntools/timeseries_products/aggregated_timeseries.py

```diff
@@ -15,7 +15,7 @@
 
 from aodntools import __version__
 
-TEMPLATE_JSON = resource_filename(__name__, 'aggregated_timeseries_template.json')
+TEMPLATE_JSON = resource_filename(__name__, 'gridded_timeseries_template.json')
```

Comment:
Is this an accidental change??
If the template is the same for the two products (or all three??), it should just be called "timeseries_products_template.json".

Reply:
Actually I think it will be easier to keep the templates separate. E.g. the first two products have a range of measured variables, while gridded only has temparature (and possibly salinity). 
So this needs to be changed back to 'aggregated_timeseries_template.json'

Reply:
the templates are different for each product. For the gridded is gridded_timeseries_template

Reply:
fixed